### DISCUSSION TITLE
Passing User Variables to User Proxy Agent - initiate_chat

### DISCUSSION THREAD

**Discussion by kuppan4109** [microsoft/autogen#2862] [Q&A]
I'm working on a code - where I need to pass chat_id ( received from fastapi request parameters) to initiate_chat so that other functions / agents can use that and execute the functionalities 

But couldn't find anything other than message in User Proxy Agent and Initiate chat 

Suggest some options to pass variables between multi-agents which is received from the API request

@sonichi @ekzhu  your answers are welcomed

**Comment by Josephrp** [comment]
well i'm not a top class engineer from microsoft but you can take user input and pass it to a variable then pass the variable in system messages for example using `f"today's date is {date} the user's name is {name} your task is {task}, do your thing:"` that's what i'm doing basicaly , hope this helps !
